### PR TITLE
[TEVA-1901] variants for upcoming cookie banner ab test

### PR DESCRIPTION
**Pull request by alexbowen**
https://dfedigital.atlassian.net/browse/TEVA-1901

### CODE REVIEW

File: app/components/test/cookies_banner_component.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Test::CookiesBannerComponent < ViewComponent::Base
```
Comment (cpjmcquillan) [inline app/components/test/cookies_banner_component.rb:1]: Can we go with AbTest instead please? Makes it clearer what these components are about imo

File: app/components/test/cookies_banner_component.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  attr_accessor :utm_parameters
```
Comment (cpjmcquillan) [inline app/components/test/cookies_banner_component.rb:2]: This can be `attr_reader` I think

**Review by cpjmcquillan — APPROVED** [review]

**Comment by martin-bangoura** [comment]
Review app deployed to <https://teaching-vacancies-review-pr-2878.london.cloudapps.digital>

File: app/components/abtest/cookies_banner_component.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+class Abtest::CookiesBannerComponent < ViewComponent::Base
```
Comment (cpjmcquillan) [inline app/components/abtest/cookies_banner_component.rb:1]: Sorry, I meant AbTest, as in /ab_test/ lol

**Pull request merged**